src/Apps/Artwork/Components/TrustSignals/SecurePayment.tsx

```diff
@@ -0,0 +1,40 @@
+import { Link, LockIcon } from "@artsy/palette"
+import { SecurePayment_artwork } from "__generated__/SecurePayment_artwork.graphql"
+import React from "react"
+import { createFragmentContainer } from "react-relay"
+import { graphql } from "react-relay"
+import { TrustSignal, TrustSignalProps } from "./TrustSignal"
+
+interface SecurePaymentProps
+  extends Omit<TrustSignalProps, "Icon" | "label" | "description"> {
+  artwork: SecurePayment_artwork
+}
+
+export const SecurePayment = ({ artwork, ...other }: SecurePaymentProps) =>
```

Comment:
Implicit returns always end up having to be refactored and can be buggy. Can this component include a `return` statement?

Reply:
I'm pretty aggressive in never having `return` statements if I can help it 😄.

I get the sentiment and reasoning though. This should just be a lint rule.

Reply:
It really is a headache for future devs. They always get unwound. For example, if a user needed to add a console log, or a new variable, or any number of things. Its considerate.

Reply:
Totally, I get it. Personally I lean heavily on vscode's refactoring feature, but I understand not everyone uses that. I'll update it.

Reply:
Thank you 👍